server/src/score/ro/raw-data.ro.ts

```diff
@@ -0,0 +1,33 @@
+import { WorkshopScore } from '../workshop-score.entity';
+
+const roHeaders = {
+  sheet: 'Raw Data',
+  columns: [
+    { label: 'Facility Name ', value: 'facilityName' },
+    { label: 'Project', value: 'projectTitle' },
+    { label: 'Estimated Project Cost', value: 'totalEstimatedCost' },
+    { label: 'Raw Ask', value: 'asks' },
+    { label: 'Overall Score', value: 'finalScore' },
+    { label: 'Score Ratio', value: 'scoreRatio' },
+  ],
+};
+
+export class RawDataRo {
+  result: any;
+  constructor(data: WorkshopScore[]) {
+    this.result = [{ ...roHeaders, ...this.convertWorkshopScoreToContent(data) }];
+  }
+  convertWorkshopScoreToContent(data: WorkshopScore[]) {
+    const content = data.map((item) => {
+      return {
+        facilityName: item.application.facilityName,
```

Comment:
Refer application-score.ro.ts -> Facility name and project title should be retrieved from the submission!